packages/kit/src/core/create_app/index.js

```diff
@@ -21,11 +25,11 @@ export function write_if_changed(file, code) {
 /** @typedef {import('types/internal').ManifestData} ManifestData */
 
 /**
- * @param {{
- *   manifest_data: ManifestData;
- *   output: string;
- *   cwd: string;
- * }} options
+ * Updates or creates two files - manifest.js and root.svelte - in the specified output directory.
+ * @param {Object} options
```

Comment:
I don't think this is correct. There aren't four parameters, but a single parameter. See https://stackoverflow.com/a/31573441 for suggestions

Reply:
Ah, sorry if the `options.` was already there. I think I missed that on my first time around